opencti-platform/opencti-graphql/src/modules/playbook/playbook-components.ts

```diff
@@ -340,11 +347,36 @@ const PLAYBOOK_CONTAINER_WRAPPER_COMPONENT: PlaybookComponent<ContainerWrapperCo
     if (container_type && isStixDomainObjectContainer(container_type)) {
       const baseData = extractBundleBaseElement(dataInstanceId, bundle);
       const created = baseData.extensions[STIX_EXT_OCTI].created_at;
-      const containerData = {
+      const containerData: Record<string, unknown> = {
         name: extractStixRepresentative(baseData) ?? `Generated container wrapper from playbook at ${created}`,
         created,
         published: created,
       };
+      // Adapt the container with generated mandatory fields, until we have some mappings in the UI
+      if (container_type === ENTITY_TYPE_CONTAINER_NOTE) {
+        containerData.content = `Generated Note content from playbook at ${created}`;
+      }
+      if (container_type === ENTITY_TYPE_CONTAINER_OBSERVED_DATA) {
+        // "objects" cannot be empty, and we need the standard_id for calling generateStandardId on the containerData ("objects" resolver in identifier.js)
+        containerData.objects = [{ standard_id: dataInstanceId }];
+        containerData.first_observed = created;
+        containerData.last_observed = created;
+        containerData.number_observed = 1;
+      }
+      if (container_type === ENTITY_TYPE_CONTAINER_OPINION) {
+        containerData.explanation = `Generated Opinion explanation from playbook at ${created}`;
+        containerData.opinion = 'neutral';
+      }
+      // if (container_type === ENTITY_TYPE_CONTAINER_REPORT) -- name is enough
+      if (container_type === ENTITY_TYPE_CONTAINER_GROUPING) {
+        containerData.context = 'unspecified';
+      }
+      // if (container_type === ENTITY_TYPE_CONTAINER_CASE_INCIDENT) -- name is enough
+      // if (container_type === ENTITY_TYPE_CONTAINER_CASE_RFI) -- name is enough
+      // if (container_type === ENTITY_TYPE_CONTAINER_CASE_RFT) -- name is enough
+      // if (container_type === ENTITY_TYPE_CONTAINER_FEEDBACK) -- name is enough
+      // if (container_type === ENTITY_TYPE_CONTAINER_TASK) -- name is enough
+      // if (container_type === ENTITY_TYPE_CONTAINER_CASE) -- stix object can be created, but cannot be ingested by opencti (abstract)
```

Comment:
I think we shall prevent users from selecting this type of container.
@richard-julien 'Case' are not part of stix v2, am I right ? Should it be made Abstract in our model ? This cannot be created in opencti actually.